stories/examples/RangeSlider.js

```diff
@@ -11,6 +12,19 @@ const wrapper = (props = {}) => {
     value,
   } = props
 
+  if ( props.type === 'sqft' ) {
+    return (
+      <DisplayTextSlider
```

Comment:
any reason you decided to go with a wrapped component instead of giving the existing rangeslider component a way to support static labels?